### DISCUSSION TITLE
Recommended bundled maps based on selected genre preference

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#18198] [Ideas]
i found a suggestion today that caught my interest

![image](https://user-images.githubusercontent.com/36758269/167522531-c1b445e8-c89e-442c-a944-4ba988e70eec.png)

so basically you'd select what genres you like and the game downloads bundled songs based on that

**Comment by Ch3mson** [comment]
Yeah, people should be able to package songs that people can download in bulk

**Comment by kstefanowicz** [comment]
There are the existing [beatmap packs](https://osu.ppy.sh/beatmaps/packs) - I'd imagine those could be made available to download in-client

**Reply by Ch3mson** [reply]
Yeah, but the beatmap packs if downloaded off the web usually have duplicate songs. Maybe the game its self can filter those out of it's downloaded through the client.
